src/components/Tabs.react.js

```diff
@@ -326,8 +480,10 @@ Tabs.defaultProps = {
         background: '#f9f9f9',
     },
     vertical: false,
+    next_tab: false,
     persisted_props: ['value'],
     persistence_type: 'local',
+    next_tab_layout: {title: 'Next', style: null},
```

Comment:
`next_tab` and `next_tab_layout` could be combined into a single `navigation` prop that would be extensible in the future. For example:

```
navigation: PropTypes.object({
  next: PropTypes.oneOf([
    true,
    PropTypes.object({
      title: PropTypes.string,
      style: PropTypes.object
    })
  ])
})
```

Using `true` would default to
```
{ title: 'Next', style: undefined }
```

Defining `title`, `style` would combine/override the defaults above.

Reply:
Later we can add `Previous`, `Last`, `First`, `Hide Tabs`, etc. to suit our needs.

Reply:
Instead of `title`, let's do `label` so that it matches `dcc.Tab` (https://github.com/plotly/dash-core-components/blob/3bcc2384576bfc7e9ae12a166f1e38615d9a2c8d/src/components/Tab.react.js#L18-L21)